docs/manual/external-tools.tex

```diff
@@ -537,6 +538,34 @@ \section{Javac compiler\label{javac-installation}}
 % \end{itemize}
 
 
+\section{Lombok\label{lombok}}
+
+Project Lombok (\url{https://projectlombok.org/}) is ``a java library that
+automatically plugs into your editor and build tools, spicing up your java.''
+Starting with version 1.18.3 it is generally easy to combine Lombok and the
+Checker Framework, because type annotations are e.g. copied from fields to
```

Comment:
When Lombok generates code, that code contains type annotations.

Reply:
Is that a suggested alternative to the `because...` part?

Reply:
Yes, that is relevant text that should be clarified.